iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/bucket/DocOrgRelationAffOrgJoiner.java

```diff
@@ -0,0 +1,48 @@
+package eu.dnetlib.iis.wf.affmatching.bucket;
+
+import org.apache.spark.api.java.JavaPairRDD;
+import org.apache.spark.api.java.JavaRDD;
+
+import eu.dnetlib.iis.wf.affmatching.bucket.projectorg.model.AffMatchDocumentOrganization;
+import eu.dnetlib.iis.wf.affmatching.model.AffMatchAffiliation;
+import eu.dnetlib.iis.wf.affmatching.model.AffMatchOrganization;
+import scala.Tuple2;
+
+/**
+ * Implementation of {@link AffOrgJoiner} that joins {@link AffMatchAffiliation} with {@link AffMatchOrganization}
+ * based on document-organization relations.
+ * 
+ * @author madryk
+ */
+public class DocOrgRelationAffOrgJoiner implements AffOrgJoiner {
+
+    private static final long serialVersionUID = 1L;
+
+
+    //------------------------ LOGIC --------------------------
+
+    /**
+     * Joins the given affiliations with organizations based on document-organization relations.<br />
```

Comment:
It might be worth explaining where those document-organization relations come from and why they indicate given affiliation organization pair is valid.